app/modules/import-export/module.php

```diff
@@ -601,16 +615,24 @@ private function get_config_data() {
 		$export_nonce = wp_create_nonce( 'elementor_export' );
 		$export_url = add_query_arg( [ '_nonce' => $export_nonce ], Plugin::$instance->app->get_base_url() );
 
-		return [
+		$config_data = [
 			'exportURL' => $export_url,
 			'summaryTitles' => $this->get_summary_titles(),
 			'builtinWpPostTypes' => ImportExportUtils::get_builtin_wp_post_types(),
 			'elementorPostTypes' => ImportExportUtils::get_elementor_post_types(),
 			'isUnfilteredFilesEnabled' => Uploads_Manager::are_unfiltered_uploads_enabled(),
 			'elementorHomePageUrl' => $this->get_elementor_home_page_url(),
 			'recentlyEditedElementorPageUrl' => $this->get_recently_edited_elementor_page_url(),
+			'tools_url' => Tools::get_url(),
 			'importSessions' => Revert::get_import_sessions(),
 		];
+
+		$last_import_session = $this->revert->get_last_import_session();
+		if ( ! empty( $last_import_session ) ) {
+			$config_data['lastImportedSession'] = $last_import_session;
+		}
+
```

Comment:
maybe just always return it? 
so on the client side you won't have to check each time for the key existence